spec/acceptance/policy_spec.rb

```diff
@@ -21,14 +21,16 @@ class { '::rabbitmq':
         ensure => present,
       } ->
 
-      rabbitmq_policy { 'ha-all@myhost':
-        pattern    => '.*',
-        priority   => 0,
+      rabbitmq_policy { 'ha-exactly@myhost':
+        pattern    => 'ha_.*',
+        priority   => 1,
         applyto    => 'all',
-        definition => {
-          'ha-mode'      => 'all',
-          'ha-sync-mode' => 'automatic',
-        },
+        # Workaround for Puppet interpreting numbers as strings
```

Comment:
This will only apply to puppet < 4, because as of 4.0.0 integers will not be integers.

Should we instead munge strings of `^\d+$` to integers?